src/pgduckdb_metadata_cache.cpp

```diff
@@ -328,7 +324,7 @@ IsMotherDuckTable(Relation relation) {
 
 bool
 IsMotherDuckEnabled() {
-	return IsMotherDuckEnabledAnywhere() && IsMotherDuckPostgresDatabase();
+	return IsMotherDuckEnabledAnywhere();
```

Comment:
Maybe just make `IsMotherDuckEnabled` and `IsMotherDuckEnabledAnywhere` a single now.